docs/_layouts/global.html

```diff
@@ -207,14 +205,79 @@ <h1 class="title">{{ page.title }}</h1>
     })
 </script>
 
+<!-- Pure JavaScript implementation of ScrollSpy -->
 <script>
-    var controller = new ScrollMagic.Controller();
+    //finds toc and automatically sorts by main list items as parents and nested list items as children
```

Comment:
Please add a space after the "//". This applies to other comments in this javascript block.